packages/stylable.io/docs/references/imports.md

```diff
@@ -6,39 +6,95 @@ layout: docs
 
 **Stylable** enables you to import other stylesheets and modules in a way that is similar to [JS Imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import). You can then use the stylesheet or module as it's been defined, or just one or more of its named values, in your own **Stylable** stylesheet.
 
-You use the **Stylable** syntax beginning with `-st-` for the `:import` config:
+## Import syntax
+**Stylable** supports two types of import syntax that can be used, the two are identical in their capabilities.
```

Comment:
Maybe arrange into:

- import definitions separated from specific syntax: `from`, `default` & `named`
- supported modules short explanation with links for more details
- syntax: new and legacy (keep terminology identical between them)